frontend/src/App.js

```diff
@@ -37,6 +39,16 @@ class App extends Component {
       .catch(() => {
         this.setState({ isLoggedIn: false, user: this.guestUser });
       });
+
+    actions.checkLocalStorage().then(loaded => {
+      this.setState({ cmsReady: loaded });
+      if (loaded) {
+        this.setState({
+          articles: JSON.parse(localStorage.getItem('allArticles')),
+          articleIndex: JSON.parse(localStorage.getItem('articleIndex')),
```

Comment:
should we store articleIndex in localstorage or create it after loading allArticles?